scripts/test-local.sh

```diff
@@ -1,5 +1,11 @@
 #!/usr/bin/env bash
 
+glob=$*
+if [ "$glob" == "" ]; then
+  glob="test/local test/remote"
+fi
+
 ./scripts/gen_keys.js
 ./scripts/check-i18n.js
-./scripts/tap-coverage.js test/local test/remote 2>/dev/null
+./scripts/tap-coverage.js $glob 2>/dev/null
+grunt
```

Comment:
I think this assumes `grunt-cli` is installed globally. before when it is in `scripts` npm does the lookup for you, but here you will probably need to do `node_modules/.bin/grunt`?